src/main/java/com/datastax/cdm/feature/ExtractJson.java

```diff
@@ -69,17 +69,19 @@ public boolean loadProperties(IPropertyHelper helper) {
 
 	@Override
 	protected boolean validateProperties() {
-		if ((null == originColumnName || originColumnName.isEmpty())
-				&& (null == targetColumnName || targetColumnName.isEmpty()))
+		if ((StringUtils.isBlank(originColumnName) || StringUtils.isEmpty(originColumnName))
+				&& (StringUtils.isBlank(targetColumnName) || StringUtils.isEmpty(targetColumnName)))
 			return true;
 
-		if (null == originColumnName || originColumnName.isEmpty()) {
-			logger.error("Origin column name is not set when Target ({}) are set", targetColumnName);
+		if ((StringUtils.isBlank(originColumnName) || StringUtils.isEmpty(originColumnName))
+				&& (StringUtils.isNotBlank(targetColumnName) || StringUtils.isNotEmpty(targetColumnName))) {
+			logger.error("Origin column name is not set when Target ({}) is set", targetColumnName);
 			return false;
```

Comment:
Here as well we should use isBlank & no need for checking targetColumnName as that has been verified in the previous condition